docs/concepts/feature-generation.md

```diff
@@ -3,8 +3,33 @@ User could utilize feature generation config to pre-compute and materialize the
 definition configs, to online and/or offline storage. This is a common practice when the feature transformation is 
 computation intensive.
 
+Example in Python:
+```python
+
+from datetime import datetime, timedelta
+from feathr.client import FeathrClient
+from feathr.materialization_settings import (BackfillTime,
+                                             MaterializationSettings)
+from feathr.sink import RedisSink
+
+client = FeathrClient()
+# Backfill time range is [start, end].
+backfill_time = BackfillTime(start=datetime(2020, 5, 20), end=datetime(2020, 5,20), step=timedelta(days=1))
```

Comment:
So `day` is the minimal granularity we support right?

Reply:
No. Users can provide hourly as well.